base-api.Tests/V1/Factories/EntityFactoryTest.cs

```diff
@@ -0,0 +1,20 @@
+using NUnit.Framework;
+using base_api.V1.Domain;
+using base_api.V1.Factory;
+
+namespace UnitTests.V1.Factories
+{
+    [TestFixture]
+    public class EntityFactoryTest
+    {
+        [Test]
+        public void CanBeCreatedFromUhTransactions()
```

Comment:
Could you please update the test name so it no longer mentions 'UH transactions'?

Reply:
Thanks for flagging!